CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [3.0.1](https://github.com/sonata-project/SonataAdminBundle/compare/3.0.0...3.0.1) - 2016-05-22
```

Comment:
Please do not remove the existing changelog.

Add you part on the PR body following this template: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/.github/PULL_REQUEST_TEMPLATE.md